src/test/java/co/omise/requests/AccountRequestTest.java

```diff
@@ -7,16 +7,48 @@
 import java.io.IOException;
 
 public class AccountRequestTest extends RequestTest {
+
     @Test
     public void testGet() throws IOException, OmiseException {
         Requester requester = getTestRequester();
         Request<Account> getAccountRequest = new Account.GetRequestBuilder().build();
         Account account = requester.sendRequest(getAccountRequest);
+
         assertRequested("GET", "/account", 200);
 
-        assertEquals("chakrit@omise.co", account.getEmail());
-        assertEquals("acct_4yq6tcsyoged5c0ocxd", account.getId());
         assertEquals("account", account.getObject());
         assertEquals("/account", account.getLocation());
     }
-}
+
+    @Test
+    public void testGetEmail() throws IOException, OmiseException {
+        Requester requester = getTestRequester();
+        Request<Account> getAccountRequest = new Account.GetRequestBuilder().build();
+        Account account = requester.sendRequest(getAccountRequest);
+
+        assertRequested("GET", "/account", 200);
+
+        assertEquals("chakrit@omise.co", account.getEmail());
```

Comment:
I think this assertion can move to `testGet()` it will reduce a lot of repeat code.